.github/workflows/ci.yml

```diff
@@ -11,11 +11,24 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - name: Run tests
-        run: |
-          PATH="$PATH:./node_modules/.bin/"
-          npm install
-          npm install spago@next purescript
-          npx playwright install --with-deps chromium
-          spago test
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18
+      - run: npm install
+      - run: npm install purescript
+      - run: npx playwright install --with-deps chromium
+      - run: npx spago@next test
+
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18
+      - run: npm run format
+      - name: Commit changes
+        uses: stefanzweifel/git-auto-commit-action@v5
+        with:
+          commit_message: Apply formatting changes
```

Comment:
I'm not a fan of automatically committing formatting changes - could we just run the autoformatter in check mode here?
There is `purs-tidy check`, and I guess prettier has a similar mode